### DISCUSSION TITLE
Ordering of folders

### DISCUSSION THREAD

**Discussion by kvalev** [photoprism/photoprism#894] [Q&A] [answered]
I am having one small issue with the ordering of the folders in UI and I hope you can help me understand why this is happening.
I have two albums, which in my opinion are sorted incorrectly:
- the first one, named `2020.12.31 - XXX` is from the new years eve celebrations and contains a bunch of photos and videos from the evening, including few pictures after midnight (so 2021-01-01)
- the second one, named `2021.01.01 - YYY` is from a completely different event from the next day

In the `folders` UI however, the album from 31.12 is shown before the other one, meaning it looks as if its "newer". I looked at the response from the REST API and both of them have the same Year/Month/Day metadata - 2021/01/01.
Is this the intended behavior - to take the metadata from the most recent photo as folder/album metadata? If it is, is there any way to ensure my folders are sorted correctly? (I assume that if the dates are equal the folders are sorted alphanumerically ascending, so in my case XXX comes after YYY).

**Comment by lastzero** [comment, marked as answer]
Since albums with the same date will be sorted by name: Would it be possible to rename one of the albums so that the order matches your expectations? We currently don't store the exact time of albums, thus the order based on dates is the same.

**Reply by kvalev** [reply]
I considered that, but I would prefer a "broader" solution, which can also help others. From the looks of it there are two options:
- allow the user to change the date (UI/REST API)
- allow the user to configure how the date should be calculated (based on the first/last photo etc), I think I saw an issue for that, but cant find it anymore

Which option would you prefer (or maybe there is a another, better one)? I can contribute the feature, that's not a problem.

**Reply by lastzero** [reply]
We'll get back to this once we're done with high priority tasks on your roadmap: https://github.com/photoprism/photoprism/projects/5
